packages/react/src/components/footer/Footer.tsx

```diff
@@ -45,69 +41,48 @@ export type FooterProps = React.PropsWithChildren<{
   title?: React.ReactNode;
```

Comment:
The comment above is most likely copied from header. The title isn't actually next to logo in footer.
Should be e.g. "The title of the service shown on the top of the footer."